src/tsickle_compiler_host.ts

```diff
@@ -141,7 +141,6 @@ export class TsickleCompilerHost implements ts.CompilerHost {
       fileName: string, content: string, writeByteOrderMark: boolean,
       onError?: (message: string) => void, sourceFiles?: ts.SourceFile[]): void {
     if (path.extname(fileName) !== '.map') {
-      fileName = this.delegate.getCanonicalFileName(fileName);
```

Comment:
Is there a test that this affects?

I think you should check this against g3 before you submit, because I think this was important for blaze-bin vs blaze-genfiles